tests/framework/installer/ceph_manifests.go

```diff
@@ -2256,7 +2256,7 @@ users:
 }
 
 // GetRookOperator returns rook Operator manifest
-func (m *CephManifestsMaster) GetRookOperator(namespace string) string {
+func (m *CephManifestsMaster) GetRookOperator(systemNamespace string) string {
```

Comment:
in the same spirit as the other comment above...
```suggestion
func (m *CephManifestsMaster) GetRookOperator(operatorNamespace string) string {
```